google/resource_compute_instance.go

```diff
@@ -378,22 +378,39 @@ func resourceComputeInstance() *schema.Resource {
 
 			"scheduling": &schema.Schema{
 				Type:     schema.TypeList,
+				MaxItems: 1,
 				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					// The `scheduling` block is optional in the config but implicitly has a length
+					// of 1 with the default values for `automatic_restart` and `preemptible`.
+					// This suppress the diff `scheduling.#: "1" => "0"` when the `scheduling` block
+					// is not specified by the user.
+					if k == "scheduling.#" {
```

Comment:
Is there a possibility of this over-suppressing? Like if an element is added/removed/changed but `scheduling.#` is still 1? Why not `Computed`?